profiler/src/ProfilerEngine/Datadog.Profiler.Native/ContentionProvider.cpp

```diff
@@ -17,16 +17,33 @@ ContentionProvider::ContentionProvider(
     IFrameStore* pFrameStore,
     IThreadsCpuManager* pThreadsCpuManager,
     IAppDomainStore* pAppDomainStore,
-    IRuntimeIdStore* pRuntimeIdStore)
+    IRuntimeIdStore* pRuntimeIdStore,
+    IConfiguration* pConfiguration)
     :
     CollectorBase<RawContentionSample>("ContentionProvider", pThreadsCpuManager, pFrameStore, pAppDomainStore, pRuntimeIdStore),
     _pCorProfilerInfo{pCorProfilerInfo},
-    _pManagedThreadList{pManagedThreadList}
+    _pManagedThreadList{pManagedThreadList},
+    _sampler(pConfiguration->ContentionSampleLimit(), pConfiguration->GetUploadInterval().count() * 1000),
```

Comment:
use the time unit and leave the computation in later

Reply:
✅ done